pkg/types/ccipocr3/config.go

```diff
@@ -12,12 +16,6 @@ type CommitPluginConfig struct {
 	// DestChain is the ccip destination chain configured for the commit plugin DON.
 	DestChain ChainSelector `json:"destChain"`
 
-	// FChain defines the FChain value for each chain. FChain is used while forming consensus based on the observations.
-	FChain map[ChainSelector]int `json:"fChain"`
-
-	// ObserverInfo is a map of oracle IDs to ObserverInfo.
-	ObserverInfo map[commontypes.OracleID]ObserverInfo `json:"observerInfo"`
-
 	// PricedTokens is a list of tokens that we want to submit price updates for.
 	PricedTokens []types.Account `json:"pricedTokens"`
```

Comment:
I think this should be part of ObservationConsensus struct not the plugin config. It seems that it will be changing and we need to reach consensus on it between observers in DON.
CC @dimkouv @makramkd